scripts/ci/run-integration-tests

```diff
@@ -6,48 +6,59 @@
 # concept
 
 set -e
-shopt -s globstar # to allow ** in globs
 
 PROJECT_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )/../../" && pwd )"
 RETRIES=3
 
-function executeCypress {
```

Comment:
I think having a function here was more readable, but feel free to overrule me :)

Reply:
hm, yeah… i can revert this if you like 👍